### PR TITLE
AM-457: Case Document AM API-Download - document binary content functional tests

### PR DESCRIPTION
**Pull request by SivaKanukolluQA**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-457

### Change description ###
Implementing functional tests AM-457: Case Document AM API-Download - document binary content.

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: src/functionalTest/resources/features/F-002/S-023.td.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "expectedResponse": {
+    "_extends_": "Common_400_Response",
+    "headers": {
+      "Connection" : "close",
+      "Content-Type" : "application/json"
+    },
+    "body" : {
+      "timestamp" : "[[ANYTHING_PRESENT]]",
+      "status" : 400,
+      "error" : "Bad Request",
```
Comment (MSancaktutar) [inline src/functionalTest/resources/features/F-002/S-023.td.json]: These 2 line looks redundant, as they are already inherited from Common_400_Response.

File: src/functionalTest/resources/features/F-002/S-022.td.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "_extends_": "Common_401_Response",
+    "headers": {
+      "Connection" : "keep-alive",
+      "Content-Type" : "application/json",
+      "Keep-Alive" : "[[ANYTHING_PRESENT]]"
+    },
+    "body" : {
+      "timestamp" : "[[ANYTHING_PRESENT]]",
+      "status" : 401,
+      "error" : "Unauthorized",
```
Comment (MSancaktutar) [inline src/functionalTest/resources/features/F-002/S-022.td.json]: These 2 line looks redundant, as they are already inherited from Common_401_Response.

File: src/functionalTest/resources/features/F-002/S-021.td.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "_extends_": "Common_404_Response",
+    "headers": {
+      "Connection" : "keep-alive",
+      "Content-Type" : "application/json",
+      "Keep-Alive" : "[[ANYTHING_PRESENT]]"
+    },
+    "body" : {
+      "timestamp" : "[[ANYTHING_PRESENT]]",
+      "status": 404,
+      "error": "Not Found",
```
Comment (MSancaktutar) [inline src/functionalTest/resources/features/F-002/S-021.td.json]: These 2 line looks redundant, as they are already inherited from Common_404_Response.

File: src/functionalTest/resources/features/F-002/S-021.td.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    "pathVariables": {
+      "documentId": "00000000-0000-0000-0000-000000000000"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_404_Response",
+    "headers": {
+      "Connection" : "keep-alive",
+      "Content-Type" : "application/json",
```
Comment (MSancaktutar) [inline src/functionalTest/resources/features/F-002/S-021.td.json]: these may also be inherited. pls double-check.

File: src/functionalTest/resources/features/F-002/S-021.td.json (hunk 3 above)
Comment (SivaKanukolluQA) [reply src/functionalTest/resources/features/F-002/S-021.td.json]: yes. These two are inherited and removed from response body.

File: src/functionalTest/resources/features/F-002/S-021.td.json (hunk 4 above)
Comment (SivaKanukolluQA) [reply src/functionalTest/resources/features/F-002/S-021.td.json]: I could not find these headers in following common headers:
Common_404_Response
Common_Response_Headers

File: src/functionalTest/resources/features/F-002/S-022.td.json (hunk 2 above)
Comment (SivaKanukolluQA) [reply src/functionalTest/resources/features/F-002/S-022.td.json]: Yes, it's inherited. Removed from response.

File: src/functionalTest/resources/features/F-002/S-023.td.json (hunk 1 above)
Comment (SivaKanukolluQA) [reply src/functionalTest/resources/features/F-002/S-023.td.json]: Yes, inherited from Common_400_Response_Body.
Removed from response body.

**Pull request merged**